robot-core/src/main/resources/report_queries/missing_subset_declaration.rq

```diff
@@ -1,6 +1,6 @@
 # # Missing Subset Declaration
 #
-# **Problem:** A synonym type is used in an annotation, but is not properly declared as a child of oboInOwl:SynonymTypeProperty. This can cause problems with conversions to OBO format.
+# **Problem:** A subset is used in an annotation (via inSubset), but is not properly declared as a child of oboInOwl:SubsetProperty. This can cause problems with conversions to OBO format, and should be avoided as all subsets should have metadata
```

Comment:
```suggestion
# **Problem:** A subset is used in an annotation (via oboInOwl:inSubset), but is not properly declared as a child of oboInOwl:SubsetProperty. This can cause problems with conversions to OBO format, and should be avoided as all subsets should have metadata.
```